package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "ginue",
-  "version": "2.1.3",
+  "version": "2.1.4",
```

今回はバグ修正じゃなくて機能追加なので、
`patch`よりも`minor`バージョンを上げたいんですが、どう思います？
https://semver.org/lang/ja/
```suggestion
  "version": "2.2.0",
```

---

確かに！ 👍